build/flatpak/com.authormore.Penpot-Desktop.metainfo.xml

```diff
@@ -0,0 +1,174 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2024 Berk Elyesa Yıldırım <beryesa@sayafe.org> -->
+<component type="desktop-application">
+    <id>com.authormore.Penpot-Desktop</id>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>AGPL-3.0-or-later</project_license>
+    <name>Penpot Desktop</name>
+    <summary>Design and code beautiful products together</summary>
+    <developer id="com.authormore">
+        <name>Belar</name>
+    </developer>
+    <description>
+        <p>
+            Penpot Desktop is an unofficial desktop application for the open-source UI/UX design tool, Penpot.
+        </p>
+        <p>
+          It delivers a desktop-like experience for Penpot users with the addition of integrated tabs to 
+          conveniently traverse back and forth between projects. Offline support is available through the 
+          &quot;select your own instance&quot; option, and theme settings can be applied to either the 
+          entire desktop app or just the Penpot dashboard.
+        </p>
+    </description>
+    <launchable type="desktop-id">com.authormore.Penpot-Desktop.desktop</launchable>
+    <provides>
+        <mediatype>application/x-penpot</mediatype>
+    </provides>
+    <branding>
+        <color type="primary" scheme_preference="light">#dde3e9</color>
+        <color type="primary" scheme_preference="dark">#34393c</color>
+    </branding>
+    <content_rating type="oars-1.1"/>
+    <url type="homepage">https://www.penpot.app</url>
+    <url type="help">https://help.penpot.app/</url>
+    <url type="bugtracker">https://github.com/author-more/penpot-desktop.git</url>
+    <screenshots>
+        <screenshot type="default">
+            <image>https://github.com/author-more/penpot-desktop/blob/flatpak/screenshots/penpot-1.png?raw=true</image>
```

Comment:
this should reference a specific commit. Never the tip of a branch, because that changes.

Reply:
Wouldn't it be nice to update when it's updated?